docs/template/public/less/_variables.less

```diff
@@ -22,4 +22,11 @@
 @dark-blue: #27394f;
 @side-nav-link: #dbe3eb;
 @top-nav-hover: #0376C6;
-@btn-type-color: #0b0080;
\ No newline at end of file
+@btn-type-color: #0b0080;
+@kss-green: #04ce26;
```

Comment:
@HybrisRawls @taifunhaeberle  - isn't this file only for kss site? I talked to jeremy about this and he said not to import frame work variables here to keep things simple.